csharp/json/src/JsonSerializerException.cs

```diff
@@ -0,0 +1,23 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+
+namespace Microsoft.Azure.Kafka.SchemaRegistry.Json
+{
+    using System;
+
+    public class JsonSerializerException : Exception
```

Comment:
Does Kafka have a .Net version of SerializationException that we can use instead of the generic System.Exception?

Reply:
There is a KafkaException that throws errors based on an ErrorCode enum, two options of which are value serialization and value deserialization errors. Only thing is it doesn't take strings in its constructor, so it relies purely on the error code for reporting the nature of the exception.
https://docs.confluent.io/platform/current/clients/confluent-kafka-dotnet/_site/api/Confluent.Kafka.KafkaException.html#constructors